BarcodeParserBuilder/Exceptions/GS1/Code128ParseException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BarcodeParserBuilder.Exceptions.GS1
```

Comment:
these exceptions should be in the Code128 namespace, not GS1

Reply:
Yes, right. I moved rest of the code but forgot exceptions. Will do!